<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="MainMenuWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#MainMenuWidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(179, 204, 117, 255), stop:1 rgba(255, 255, 255, 255))};</string>
   </property>
   <widget class="QLabel" name="menuGlowneTekst">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;Menu główne&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="wyborOperacjiTekst">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Wybierz operację!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modelHaty_przycisk">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Model Haty</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rachunek_db_przycisk">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Rachunek decybelowy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="przycisk3_PrawoOhma">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>190</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Obwody elektryczne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operacja4_ONP">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Notacja Polska</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rownanie_friisa_przycisk">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>390</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Równanie Friisa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operacja_6">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>390</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Operacja6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operacja_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>570</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Operacja7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operacja_8">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>570</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Operacja8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operacja_9">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>570</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Operacja9</string>
    </property>
   </widget>
   <widget class="QLabel" name="menuQuote">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>670</y>
      <width>800</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="wordWrap">
           <bool>true</bool>
          </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;left&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:300; font-style: italic;&quot;&gt;QUOTE HERE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="menuQuoteAuthor">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>750</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="wordWrap">
           <bool>true</bool>
          </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:200;&quot;&gt;QUOTE HERE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
